README.md

```diff
@@ -25,7 +25,7 @@ The module supports the following operations:
 ## Overview
 * [terraform-ibm-secrets-manager-iam-engine](#terraform-ibm-secrets-manager-iam-engine)
 * [Examples](./examples)
-    * [Secret Manager and IAM Secrets Engine setup](./examples/default)
+    * [Default example](./examples/default)
```

Comment:
I would go with Basic example to be consistent with other modules we have onboarded